packages/stylelint/test/middleware_test.js

```diff
@@ -0,0 +1,37 @@
+import test from 'ava';
+import { Neutrino } from 'neutrino';
+
+const mw = () => require('..');
+const options = { stylelint: { rules: { 'rule-empty-line-before': true } } };
+
+test('loads middleware', t => {
+  t.notThrows(mw);
+});
+
+test('uses middleware', t => {
+  t.notThrows(() => Neutrino().use(mw()));
+});
+
+test('uses with options', t => {
+  t.notThrows(() => Neutrino().use(mw(), options));
+});
+
+test('instantiates', t => {
+  const api = Neutrino();
+
+  api.use(mw());
+
+  t.notThrows(() => api.config.toConfig());
+});
+
+test('instantiates with options', t => {
+  const api = Neutrino();
+
+  api.use(mw(), options);
+
+  t.notThrows(() => api.config.toConfig());
+});
+
+test('exposes stylelintrc config', t => {
+  t.is(typeof Neutrino().use(mw()).call('stylelintrc'), 'object');
+});
```

Comment:
Hmmm - This last test causes tests to fail: https://travis-ci.org/mozilla-neutrino/neutrino-dev/jobs/316091388#L920-L932 not sure why…seems identical to the eslintrc test…

Reply:
Run the tests locally and see the output of `Neutrino().use(mw()).call('stylelintrc')`.

Reply:
Oh derp - it's b/c stylelint throws err if you don't pass it anything.
Fixed.